coops-ui/src/views/AnnualReport.vue

```diff
@@ -503,6 +506,16 @@ export default {
       this.toggleFiling(modified ? 'add' : 'remove', 'OTFDR')
     },
 
+    getSectionCode () :string {
```

Comment:
I recommend you move this function (and the ones in the other 2 filings below) to the mixin as well. I realized they don't need to be shared by different components but it would keep all this logic and data in one place.

Reply:
I'd love to, but the section code actually changes based on which path we are in (AR vs Standalone Director change etc.), unless I were to make a switch-case of switch-cases and pass the path in as a parameter

Reply:
Could the functions be named differently?

Reply:
Yeah I mean I don't see a clean way of doing this without a lookup of some form...I'd imagine we have to revisit this, since every entity potentially has different section codes (and I don't think we wants hundreds of lines of code just for this)